pkg/controller/controller_instance.go

```diff
@@ -220,6 +221,17 @@ func (c *controller) reconcileInstance(instance *v1alpha1.Instance) error {
 				successProvisionMessage,
 			)
 			c.recorder.Eventf(instance, api.EventTypeNormal, successProvisionReason, successProvisionMessage)
+		} else {
+			// the broker returned a failure response
+			errorProvisionCalledMessage := fmt.Sprintf("provision call failed")
+			c.updateInstanceCondition(
```

Comment:
fwiw, i notice a consistent pattern where we ignore any error that can be returned from this.

i think we should either return, or at minimum log the error.